requirements.txt

```diff
@@ -1,5 +1,7 @@
-pyrovision@git+https://github.com/pyronear/pyro-vision.git#egg=pyrovision
 pyroclient@git+https://github.com/pyronear/pyro-api.git#egg=pyroclient&subdirectory=client
 python-dotenv==0.15.0
 requests==2.25.1
 psutil==5.9.0
+torch
+torchvision
+opencv-python
```

Comment:
please pin those versions :pray: 

Do we need openCV by the way?

Reply:
yes we do for know because we are using ultralytics to load the model but I will fix this in an upcoming PR ;)